module4/projects/employer_case_study/expectations.md

```diff
@@ -5,83 +5,127 @@ layout: page
-
-**IMPORTANT: You do NOT need to wait until after your interview simulation to complete these steps or apply to the job.**
+## Part 2: The Job Application
+The next step of the employer case study is to apply to your chosen job opportunity. 
 
-### Application Process:
+<section class="note">
+**IMPORTANT:** You do NOT need to wait until after your interview simulation to complete these steps or apply to the job. If you'd like feedback on your materials sooner than the interview simulation, you can schedule a check-in with your instructor.
+</section>
 
-**Outreach to Someone at the Company**
+### Application Process
+<section class="dropdown">
+### 1. Outreach to Someone at the Company
 - Draft effective outreach messages, which could be emails, LinkedIn messages, or other suitable platforms.
 - Apply strategies to capture the recipient's attention, showcase their genuine interest in the company, and encourage a positive response.
+</section>
 
-**Prepare for the Meeting**
+<section class="dropdown">
+### 2. Prepare for the Meeting
 - If the outreach leads to a response and a meeting is scheduled, prepare for it by doing things like:
   * Developing a list of questions and topics to discuss
   * Researching the person you'll meet with
   * Considering your own value proposition.
+</section>
 
-**Complete Application**
+<section class="dropdown">
+### 3. Complete Application
 - After networking and meeting with someone from the company, complete the job application process. You can further tailor your resume and cover letter to reflect the insights gained during your research and discussions.
   - **NOTE:** If you were unable to connect with someone at the company, you should still tailor your resume and cover letter with the details you learned during the Employer Persona Research Phase.
 - Follow through with the application, considering any additional materials or follow-up steps as needed.
-
+</section>
 
 This comprehensive job search process allows you to take the knowledge and skills you've acquired through the employer persona research and apply them to real-world job applications. It ensures that you approach their job search strategically and professionally, increasing your chances of success in securing roles that align with your career goals and values.
 
 <!-- For additional information and resources on how to find job opportunities and contacts, see the [4-Step Job Application Process Document](https://docs.google.com/document/d/1B39ViRamGHNVLzelnlEIhzLWNwsrPvf1J8MjtxDKgsc/edit#heading=h.ncblao9s0orb). -->
-### Part 4 Deliverables
-* Outline your application process, network efforts, and the outcomes for each step in your [final presentation](./final_presentation.html).
-
-
-
-## Evaluation and Final Presenation
+<section class="call-to-action">
+### Part 2 Deliverables
+<!-- NOTE: Or would we rather receive this via a form some other time? Previously this was discussed as a part of the final presentation. -->
```

Comment:
I'd defer to you on what workflow makes sense. If students want on the spot feedback, there might not be time during the simulation to do that. IF this is more just an accountability mechanism, turning it in at the interview sim could be fine. My sense is the latter so I think a document is good, and then if a student has specific questions about their approach I'd hope they'd ask well before the interview simulation?